src/components/WaterCycle.vue

```diff
@@ -47,146 +46,244 @@
       </h3>
       <h3 class="optionsBar">
         Zoom:
-        <button
-          class="zoom button"
-          @click="$refs.zoomer.zoomIn()"
-        >
-          +
-        </button>
-        <button
-          class="zoom button out"
-          @click="$refs.zoomer.zoomOut()"
-        >
-          -
-        </button>
+        <button class="zoom button" @click="zoom.value = Math.min(zoom.value + 0.1, 5)">+</button>
+        <button class="zoom button out" @click="zoom.value = Math.max(zoom.value - 0.1, 1)">-</button>
```

Comment:
RC: fix zoom button behavior - currently doesn't trigger an update to the zoom level

Reply:
addressed, and also now allows panning when zoomed in or out